Samples/Swift/DaysUntilBirthday/Shared/ViewModels/VerifiedAgeViewModel.swift

```diff
@@ -32,12 +41,31 @@ final class VerifiedAgeViewModel: ObservableObject {
   /// Creates an instance of this view model.
   init() {
     self.verificationState = .unverified
+    self.ageVerificationSignal = "Signal Unavailable"
+    self.isShowingAgeVerificationSignal = false
+    self.isShowingAgeVerificationAlert = false
   }
 
   /// Verifies the user's age is over 18.
   func verifyUserAgeOver18() {
     loader.verifyUserAgeOver18()
   }
+
+  /// Fetches the age verification signal representing whether the user's age is over 18 or pending.
+  func fetchAgeVerificationSignal(verifyResult: GIDVerifiedAccountDetailResult) {
+    loader.fetchAgeVerificationSignal(verifyResult: verifyResult) {
+      self.ageVerificationSignal =
+        self.loader.verification?.signal ?? "Error: No verification details found"
+
+      if (self.ageVerificationSignal == "AGE_OVER_18_STANDARD"){
```

Comment:
For example, the enum case raw values can be things like "AGE_OVER_18_STANDARD", which would be used to get an instance of a case matching that name. It makes dealing with strings safer (i.e., more type based).

Reply:
Made enum AgeVerificationSignal to hold these values.